package.json

```diff
@@ -2,9 +2,10 @@
   "name": "leakage",
   "version": "0.2.0",
   "description": "Memory leak testing for node. Write tests using your favorite test runner.",
-  "main": "lib/index.js",
+  "main": "build/index.js",
   "scripts": {
-    "test": "standard lib/**/*.js && mocha test/"
+    "build": "babel lib -d build",
```

Comment:
`src` with `lib`, `dist` or `build` are common names.
Changed to `src` and `lib`. Thanks! 🙏